packages/orbit-components/src/TextLink/__tests__/index.test.js

```diff
@@ -62,16 +62,16 @@ describe("#TextLink", () => {
     expect(link).toHaveAttribute("rel", expect.stringContaining(rel));
     expect(link).toHaveAttribute("target", "_blank");
   });
+
   it("should no have underline and height for a11y", () => {
-    const { container } = render(
+    render(
       <TextLink noUnderline standAlone>
         {title}
       </TextLink>,
     );
-    expect(getComputedStyle(container.firstChild)).toHaveProperty("text-decoration", "none");
-    expect(getComputedStyle(container.firstChild)).toHaveProperty(
-      "height",
-      defaultTheme.orbit.heightButtonNormal,
-    );
+    expect(screen.getByRole("button")).toHaveStyle({ "text-decoration": "none" });
+    expect(screen.getByRole("button")).toHaveStyle({
+      height: defaultTheme.orbit.heightButtonNormal,
+    });
```

Comment:
Notice that you can use camelCase keys.

```suggestion
    expect(screen.getByRole("button")).toHaveStyle({
      textDecoration: "none",
      height: defaultTheme.orbit.heightButtonNormal,
    });
```

Reply:
Did you also notice the suggestion to put both CSS rules in a single object?